src/Reader/Reader.php

```diff
@@ -189,17 +197,14 @@ public static function useHttpConditionalGet($bool = true)
      */
     public static function import($uri, $etag = null, $lastModified = null)
     {
-        $cache       = self::getCache();
-        $client      = self::getHttpClient();
-        $client->resetParameters();
-        $headers = new ZendHttp\Headers();
-        $client->setHeaders($headers);
-        $client->setUri($uri);
+        $cache   = self::getCache();
```

Comment:
I have a question regarding cache here. Isn't the client is responsible for caching things?

Eg : When I am using Zend-Http-Client I will say cache this url, when using guzzle I am using https://github.com/guzzle/cache-subscriber . It looks caching seems not need in the reader. But is the responsibility of the http client to me.